src/main/java/com/revature/controllers/PersonController.java

```diff
@@ -69,23 +61,46 @@
 			 return ResponseEntity.ok(singlePerson);
 			 
 		 }
+	}
+	
+	@RequestMapping(method = RequestMethod.POST, value = "persons")
+	public ResponseEntity<Person> createPerson(@RequestParam(required=true) String firstname,
+			@RequestParam(required=true) String lastname, 
+			@RequestParam(required=true) Integer role){
 		
-	}	
+			Person person = new Person(firstname, lastname, role);
+			
+			personLogic.savePerson(person);
+
+			return ResponseEntity.ok(person);
 		
+	}
+	
+	@RequestMapping(method = RequestMethod.PUT, value = "persons")
+	public ResponseEntity<Person> modifyPerson(@RequestParam(required=true) Integer id,
+			@RequestParam(defaultValue="", required=false) String firstname,
+			@RequestParam(defaultValue="", required=false) String lastname, 
+			@RequestParam(defaultValue="0", required=false) Integer role){
+	
+			Person person = personLogic.getPersonById(id);
+			
+			System.out.println("first " + firstname + " last " + lastname + " role " + role);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)